### PR TITLE
Add unmute API endpoint to Heartbeat (CU-baj2pv)

### PR DESCRIPTION
**Pull request by trisapeace**
Added a new endpoint called `POST unmute_system` that will unmute a muted system.

To help with testing this feature (and because I think it will be useful in real life), I also added a new "Muted?" column to the Heartbeat Dashboard that will display "Y" for systems that are muted and "N" for systems that are not muted.
image

While I was in there, I also noticed that there is a `POST hide_system` endpoint but no `POST show_system`. I'm wondering if this should be added as well. What do you think @schwarrrtz ?

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-baj2pv Provide a way to un-mute heartbeat alerts through the API](https://app.clickup.com/t/baj2pv)

File: heartbeat/utils.py
Comment (trisapeace) [inline heartbeat/utils.py:26]: I just added those utility functions, @schwarrrtz .

**Comment by schwarrrtz** [comment]
This looks good to me - we're going to hold off merging this until Mario merges #40 to avoid excessive rebasing.

**Review by schwarrrtz — APPROVED** [review]

**Review by mariocimet — APPROVED** [review]

**Pull request merged**
